### PR TITLE
#70 updated the Readme file 

### PR DESCRIPTION
**Pull request by slayerjain**
I've made some necessary changes to the README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Make sure you have a nightly Rust installed. Something like this:
 
 ```bash
 $ rustc --version
-rustc 1.13.0-nightly (267cde259 2016-05-25)
+rustc 1.10.0-nightly (267cde259 2016-05-25)
```
Comment (steveklabnik) [inline README.md]: can you undo this change? 1.13 is the current nightly version. :smile:

File: README.md [hunk 2]
```diff
@@ -39,6 +39,19 @@ linker = "/Users/yourusername/opt/bin/x86_64-pc-elf-gcc"
 
 Where `yourusername` is your username.
 
+This project uses the xargo crate. To get it:
```
Comment (steveklabnik) [inline README.md]: Could we link to the project here?

File: README.md [hunk 3]
```diff
@@ -39,6 +39,19 @@ linker = "/Users/yourusername/opt/bin/x86_64-pc-elf-gcc"
...
+
+ Before install necessary dependencies, for Ubuntu 16.04, we need to install libssl-dev:
```
Comment (steveklabnik) [inline README.md]: This is probably better as a note, something like `xargo has a few dependencies, check its readme to make sure you've got them installed.

File: README.md (hunk 1 above)
Comment (WilsonGiese) [inline README.md]: 1.14 now :D

File: README.md (hunk 3 above)
Comment (WilsonGiese) [inline README.md]: This is true! `cmake` was required for me

File: README.md (hunk 1 above)
Comment (slayerjain) [inline README.md]: aren't we using a specific versions of rust compiler?

Comment (slayerjain) [inline README.md]: The version number of "267cde259 2016-05-25" 1.10

Comment (WilsonGiese) [inline README.md]: I think we were, but at some point it switched to nightly - don't remember when

Comment (slayerjain) [inline README.md]: So now any latest nightly rustc would word? :D

Comment (WilsonGiese) [inline README.md]: Should! I've been `rustup update`ing everyday and running without issues related to the rust build.

Comment (WilsonGiese) [inline README.md]: I think the version in the example is just an example to determine if you've got nightly installed, not a constraint to work against.

Comment (slayerjain) [inline README.md]: I just realised that, the earlier constraint on rustc confused me. :D

Comment (slayerjain) [inline README.md]: The mismatch between the version 1.13 and  id "267cde259", added to the confusion :D

File: README.md [hunk 4]
```diff
@@ -39,18 +39,12 @@ linker = "/Users/yourusername/opt/bin/x86_64-pc-elf-gcc"
...
 
-This project uses the xargo crate. To get it:
-
- Before install necessary dependencies, for Ubuntu 16.04, we need to install libssl-dev:
- ```bash
- $ sudo apt-get install libssl-dev
- ```
-
-Now install xargo:
+This project uses the xargo crate (https://github.com/japaric/xargo). To get it:
```
Comment (WilsonGiese) [inline README.md]: Probably should link with markup here, e.g. `[xargo](https://github.com/japaric/xargo)`

Comment (slayerjain) [inline README.md]: oh yeah!

File: README.md (hunk 1 above)
Comment (steveklabnik) [inline README.md]: Yes, @WilsonGiese is right: we used to be tied to a specific version, but with the recent changes to `xargo`, we're not any more.

**Comment by slayerjain** [comment]
i think this should be merged.

File: README.md [hunk 5]
```diff
@@ -39,6 +39,13 @@ linker = "/Users/yourusername/opt/bin/x86_64-pc-elf-gcc"
 
 Where `yourusername` is your username.
 
+This project uses the [xargo](https://github.com/japaric/xargo) crate. To get it:
+
+ ```bash
+ $ cargo install xargo
+ ```
+ Note: xargo has a few dependencies, check it's readme to make sure you've got them installed.
```
Comment (steveklabnik) [inline README.md]: can you put a newline above this line please?

Also, it's "its".

File: README.md [hunk 6]
```diff
@@ -47,6 +54,7 @@ $ make run
 
 This will:
 
+* Xargo builds libcore for our x86_64-unknown-intermezzos-gnu target
```
Comment (steveklabnik) [inline README.md]: Could you say "Build libcore for our x86_64-unknown-intermezzos-gnu target with Xargo", please?

**Comment by steveklabnik** [comment]
Thanks! Sorry I missed this, one or two more small tweaks and let's merge.

**Comment by steveklabnik** [comment]
@slayerjain ping! are you still interested in keeping up this PR?

**Comment by slayerjain** [comment]
oh yeah sorry! I'll work on this now :D

**Comment by slayerjain** [comment]
Made those changes.

**Comment by steveklabnik** [comment]
Thank you! ❤️

**Comment by slayerjain** [comment]
My pleasure! :dancing_men:
